src/vm/domainfile.cpp

```diff
@@ -761,7 +761,6 @@ BOOL DomainFile::IsZapRequired()
     if (!m_pFile->HasMetadata() || !g_pConfig->RequireZap(GetSimpleName()))
         return FALSE;
 
-#if defined(_DEBUG) && defined(FEATURE_TREAT_NI_AS_MSIL_DURING_DIAGNOSTICS)
```

Comment:
Was it intentional to not to keep the #if defined(_DEBUG)?

Reply:
Fixed